Panic when connection refused

```
panic: interface conversion: interface {} is nil, not string

goroutine 1 [running]:
myproject/vendor/github.com/TobiEiss/aranGoDriver.(*AranGoSession).Connect(0xc42000e128, 0x88c898, 0x4, 0x88cc1d, 0x5, 0x7fd812b7bbe0, 0x0)
	/path/to/myproject/vendor/github.com/TobiEiss/aranGoDriver/aranGoSession.go:38 +0x1ab
main.ArangoConnect(0x8, 0x8232e0, 0xc420110301, 0xc42000e120)
	/path/to/myproject/arangodb.go:18 +0xdc
main.init.0()
	/path/to/myproject/arangodb.go:63 +0x26
main.init()
	<autogenerated>:1 +0x322
```

Should be easy to fix, I will do a quick PR.

--- a/aranGoSession.go
+++ b/aranGoSession.go
@@ -35,7 +35,9 @@ func (session *AranGoSession) Connect(username string, password string) error {
 	credentials.Password = password
 
 	_, resp, err := session.arangoCon.Post(urlAuth, credentials)
-	session.arangoCon.SetJwtKey(resp["jwt"].(string))
+	if err == nil {
+		session.arangoCon.SetJwtKey(resp["jwt"].(string))
+	}
 	return err
 }
 

